apps/teams-test-app/src/App.tsx

```diff
@@ -17,7 +17,12 @@ const getOriginsParam = urlParams.has('origins') && urlParams.get('origins') ? u
 const validMessageOrigins: string[] | undefined = getOriginsParam ? getOriginsParam.split(',') : undefined;
 
 // This is added for custom initialization when app can be initialized based upon a trigger/click.
-if (!urlParams.has('customInit') || !urlParams.get('customInit')) {
+if (
+  !urlParams.has('customInit') ||
+  !urlParams.get('customInit') ||
+  !urlParams.has('skipInit') ||
+  !urlParams.get('skipInit')
```

Comment:
Wouldn't checking `urlParams.get('paramName') === null` be sufficient? What cases does the `.has` call also cover?